src/frontend/components/property-type/reference/edit.tsx

```diff
@@ -1,5 +1,7 @@
-import React, { ReactNode } from 'react'
-import Select from 'react-select/lib/Async'
+/* eslint-disable @typescript-eslint/no-use-before-define */
```

Comment:
You can move loadOptions on the top and this won't be needed

Reply:
I know, but it's more readable when the main thing in the file - the component - is the first thing you see when you open it. I can follow this guideline, but it might be worth considering why we need it in the first place. Hoisting is a language feature for a reason.